index/generator/library/library.go

```diff
@@ -705,6 +710,63 @@ func validateStackInfo(stackInfo schema.Schema, stackfolderDir string) []error {
 	return errors
 }
 
+// SetLastModifiedValue adds the last modified value to a pre-created index
+// The last modified dates are contained in a file named last_modified.json that is apart of the registry dir
+/* #nosec G304 -- lastModFile is produced from filepath.Join which cleans the input path */
+func SetLastModifiedValue(index []schema.Schema, registryDirPath string) ([]schema.Schema, error) {
```

Comment:
I think a test for this function is a nice addition. WDYT?

Reply:
I can add a test for that, I thought I did but maybe I overlooked it!

It won't let me reply to your comment below but I think `2023-04-08T11:51:08Z` and `2024-05-13T12:32:02+02:00` are in the same format (RFC3339 in Go, which is a strict ISO8601). The `Z` indicates UTC and the `+0200` indicates it is 2 hours offset from UTC. I believe instead of stating `+000` it shorthands it to `Z`.

Reply:
I think with the `time` pkg in go `RFC3339` would be the best format for achieving `ISO8601` but it includes the timezone (and when the `last_modified.json` file is created on the registry side it includes the timezone as well from git). I think to stick to what we talked about earlier with the server providing all the information it could be left up to the frontend to format it down without a timezone if necessary? wdyt

Reply:
> I think with the `time` pkg in go `RFC3339` would be the best format for achieving `ISO8601` but it includes the timezone (and when the `last_modified.json` file is created on the registry side it includes the timezone as well from git). I think to stick to what we talked about earlier with the server providing all the information it could be left up to the frontend to format it down without a timezone if necessary? wdyt

Yup for me is fine. It's strange from github perspective that those datetime values have such differences.